plugins/modules/network/netscaler/netscaler_nitro_request.py

```diff
@@ -642,10 +642,7 @@ def get_filtered(self):
         if self._module.params['filter'] is None:
             self.fail_module(msg='NITRO filter is undefined.')
 
-        keys = list(self._module.params['filter'].keys())
-        filter_key = keys[0]
-        filter_value = self._module.params['filter'][filter_key]
-        filter_str = '%s:%s' % (filter_key, filter_value)
+        filter_str = '&'.join('%s, %s' % (k, v) for k, v in self._module.params['filter'].items())
```

Comment:
Sorry, should have been:
```suggestion
        filter_str = '&'.join('%s:%s' % (k, v) for k, v in self._module.params['filter'].items())
```

Reply:
Ah, that's what I get for blindly copy-pasting. Good catch!